packages/governance/src/committee.js

```diff
@@ -38,8 +38,9 @@ const start = zcf => {
 
   const makeCommitteeVoterInvitation = index => {
     /** @type {OfferHandler} */
-    const offerHandler = Far('voter offerHandler', () => {
+    const offerHandler = seat => {
```

Comment:
Why remove the `Far` declaration?

Reply:
Dan asked me to justify it, and I couldn't.

I don't see other offerHandlers that are Far.  The voter facet (line 43) is still Far, and that seems to be required.